src/components/labelled-date-time-input/LabelledDateTimeInput.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DateTime from 'react-datetime'
+import moment from 'moment'
+import 'moment/locale/ru'
+import './LabelledDateTimeInput.css'
+
+const LabelledDateTimeInput = ({ label, value, timeFormat }) => {
+  return (
+    <label>
+      {label}{' '}
```

Comment:
I think we can use here `LabelledInput` component and just replace its input, using its `renderInput` prop. Please have a look at this component. It will make this component more common and reduce common code

Reply:
Please contact me, in case you need any help with applying this comment

Reply:
done